Language/Functions/Advanced IO/noTone.adoc

```diff
@@ -1,8 +1,8 @@
 ---
 title: noTone()
 title_expanded: ""
-categories: [ "Functions" ]
-subCategories: [ "Advanced I/O" ]
+categories: [ "Funzioni" ]
```

Comment:
Hi! Please do not translate 'categories', only 'subcategories'. Thanks for your work